utils/frame/try-runtime/cli/Cargo.toml

```diff
@@ -31,13 +38,6 @@ sp-weights = { version = "4.0.0", path = "../../../../primitives/weights" }
 frame-try-runtime = { optional = true, path = "../../../../frame/try-runtime" }
 substrate-rpc-client = { path = "../../rpc/client" }
 
-parity-scale-codec = "3.0.0"
-hex = "0.4.3"
```

Comment:
I think these deps were split off to the bottom since they are external.

Reply:
indeed.